reana_server/openapi_connections/reana_workflow_controller.json

```diff
@@ -454,6 +454,223 @@
         "summary": "Adds a file to the workflow workspace."
       }
     },
+    "/api/workflows/{workflow_id}/workspace/inputs": {
+      "get": {
+        "description": "This resource is expecting a workflow UUID and a filename to return its list of input files.",
+        "operationId": "get_workflow_inputs",
+        "parameters": [
+          {
+            "description": "Required. Organization which the worklow belongs to.",
+            "in": "query",
+            "name": "organization",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "description": "Required. UUID of workflow owner.",
+            "in": "query",
+            "name": "user",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "description": "Required. Workflow UUID.",
+            "in": "path",
+            "name": "workflow_id",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "produces": [
+          "multipart/form-data"
+        ],
+        "responses": {
+          "200": {
+            "description": "Requests succeeded. The list of input files has been retrieved.",
+            "schema": {
+              "items": {
+                "properties": {
+                  "last-modified": {
```

Comment:
Not to be added for this PR, but maybe checksumming (MD5, SHA1) would be a nice improvement at some point.